modules/swagger-parser/src/test/java/io/swagger/parser/SwaggerResolverTest.java

```diff
@@ -370,12 +372,11 @@ public void testSettingsAddParametersToEachOperationDisabled() {
                         "        default:\n" +
                         "          description: test response\n";
 
-        SwaggerParser parser = new SwaggerParser();
-        SwaggerDeserializationResult result = parser.readWithInfo(yaml);
 
-        Swagger swagger = result.getSwagger();
+        SwaggerDeserializationResult swaggerDeserializationResult = new SwaggerDeserializer().deserialize(DeserializationUtils.readYamlTree(yaml));
```

Comment:
Actually, using `parser.readWithInfo(yaml)` (done before) already call the SwaggerResolver, so this test was failing. Using `SwaggerDeserializer` directly allow to test `SwaggerResolver` in isolation.